.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,76 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+<!--
+Further instructions on how to fill a bug refer here:
+https://martinos.org/mne/stable/faq.html#i-think-i-found-a-bug-what-do-i-do
+
+If your issue is a usage question, submit it here instead:
+- The mne-python gitter: https://gitter.im/mne-tools/mne-python
+- The mne-analysis mailing list: https://mail.nmr.mgh.harvard.edu/mailman/listinfo/mne_analysis
+-->
+
+#### Describe the bug
+<!--
+Please provide a clear and concise description of what the bug is.
+-->
+
+#### Steps/Code to Reproduce
+<!--
+Please provide a code snippet or Minimal Working Example (MWE) to replicate your
+problem. https://en.wikipedia.org/wiki/Minimal_Working_Example
+
+This MWE should be auto conained. Other mne-python contributors should be able
+to copy paste the provided snippet and replicate your results. In this regard,
+it is prefered that the MWE use MNE sample data. Otherwise, provide all the data
+needed to replicate.
+
+If the code is too long, feel free to put it in a public gist and link
+it in the issue: https://gist.github.com
+
+Example:
```

Comment:
Can someone provide an example that resonates with MNE community?

Reply:
I'm not sure an example is necessary.

Reply:
If we want one, there is a very short, nice example in #5315 (thanks @cbrnr). I am +0.5 on adding this, since it shows people an example of SSCCE

Reply:
+0.5 too because this is such a nice MWE 😛